modules/blocks.php

```diff
@@ -190,3 +190,38 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+/**
+ * Visited Block.
+ */
+jetpack_register_block( 
+	'visited', 
+	array(
+		'render_callback' => 'jetpack_visited_block_render',
+	)
+);
+
+/**
+ * Visited block dependency declaration.
+ *
+ * @param array  $attributes - Array containing the map block attributes.
+ * @param string $content - String containing the visited block content.
+ *
+ * @return string
+ */
+function jetpack_visited_block_render( $attributes, $content ) {
+	Jetpack_Gutenberg::load_assets_as_required( 'visited' );
+
+	$count = intval( $_COOKIE[ 'wp-visit-tracking' ] );
```

Comment:
Can we use a different name for the cookie, something with a `jetpack` or `jp` prefix for example? It would make it easier for folks to understand what added the cookie in the first place.

Reply:
@jeherve: How does `jp-visit-counter` sound? I'd like to make sure I'm following any established naming conventions.

Reply:
I think that works 👍